### ISSUE TITLE
Issue when submitting multiple form input with the same name in version 1.2.1

### ISSUE THREAD

**Issue by alisaracoglu** [wintercms/winter#872]
### Winter CMS Build

1.2

### PHP Version

8.1

### Database engine

MySQL/MariaDB

### Issue description

hello. I ran into a problem submitting multiple form input with the same name in version 1.2.1. This problem does not occur in version 1.1.10.  When I add the {files:true} option to the {{ form_ajax() }} function, the array forms are sending as a single string of data. When I remove the {files:true} option, it suddenly sends as array elements as I expected. Although this is a deliberate feature, I did not see it in the documentation.
![winterIssue](https://user-images.githubusercontent.com/5839590/227463410-d27120d8-749a-41dd-9ba5-9950c817f154.jpg)

### Steps to replicate

I created a new winter project so that it is not affected by other codes. The problem occurred when I added the files:true option to the form_ajax() function. The problem did not occur when I removed the files:true option. I also did a fresh install of winter v1.1.10 to check the version. There was no problem with the same codes.

**Comment by bennothommo** [comment]
Sorry for the delay @alisaracoglu - are you using Snowboard or the original AJAX framework?

**Comment by alisaracoglu** [comment]
It's not important @bennothommo . I am using original AJAX framework.

**Comment by LukeTowers** [comment]
@bennothommo perhaps this is due to the support added for request parents?

**Comment by mjauvin** [comment]
Do you mean [this PR](https://github.com/wintercms/winter/pull/650) @LukeTowers ?

**Comment by bennothommo** [comment]
~~@alisaracoglu I've just tested this in Chrome, Edge and Firefox and I cannot replicate the behaviour. I've copied the exact same code as you, and have tried it with files enabled and disabled, and both show `categories` as an array as expected. I've also confirmed that on the PHP side it receives the input as an array both times too.~~

~~Which browser are you using?~~

EDIT: My apologies, @alisaracoglu. I thought the Demo theme in Winter still used the old framework, but it uses Snowboard. Forcing it to the old framework exhibits the issue you have reported. Looking into it now.
